recipes-graphics/wayland/weston_10.0.0.bbappend

```diff
@@ -9,7 +9,9 @@ SRC_URI:append:tegra = " file://0001-Drop-DRM-version-check-in-meson.build.patch
 DEPENDS:append:tegra = " egl-wayland"
 PACKAGE_ARCH:tegra = "${TEGRA_PKGARCH}"
 
-RDEPENDS:${PN}:append:tegra = " egl-wayland egl-gbm tegra-udrm-probeconf"
+RDEPENDS:${PN}:append:tegra = " egl-wayland egl-gbm"
+RDEPENDS:${PN}:append:tegra194 = " tegra-udrm-probeconf"
+RDEPENDS:${PN}:append:tegra234 = " nvidia-drm-probeconf"
```

Comment:
Is there a reason to keep the pattern of using a recipe to implement this? I think the tegra-udrm-probeconf might just be historical, because [`KERNEL_MODULE_AUTOLOAD`](https://docs.yoctoproject.org/dev/ref-manual/variables.html?#term-KERNEL_MODULE_AUTOLOAD) and [`KERNEL_MODULE_PROBECONF`](https://docs.yoctoproject.org/dev/ref-manual/variables.html?#term-KERNEL_MODULE_PROBECONF) are the current standard ways to accomplish this. Using the variables, we should just think if there is any case where we shouldn't be autoloading tegra-udrm/nvidia-drm, and overriding the assignment based on distro features? Certainly, if it's included in the image and loaded, it's worthless without the probeconf (`modeset=1`).

Reply:
I'm not sure about the history, I agree with you, we should use `KERNEL_MODULE_AUTOLOAD` and `KERNEL_MODULE_PROBECONF`

Reply:
I had originally enabled tegra-udrm that way in f860c21c902617cf09e67269acb10f22d55de370, but there were conflicts between it and non-Wayland/Weston DRM usage (for Qt, maybe?), so I switched to using the separate recipe that was only pulled in for W/W builds.

Assuming that conflict is gone now, we could switch back again and eliminate the extra recipe.

Reply:
Super @madisongh thanks for the explanation.
I will rework that and remove the additional recipes

Reply:
At this point Qt is fine with the tegra-udrm.